app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
```

Comment:
Assigning `Person.people[name] = self` directly in the constructor can lead to issues if multiple `Person` objects with the same name are created, as it will overwrite the previous entry. Consider using an instance variable for storing people.